server/catgenome/src/main/java/com/epam/catgenome/manager/blast/BlastTaskManager.java

```diff
@@ -159,7 +161,20 @@ public void deleteTask(final long taskId) {
     @Transactional(propagation = Propagation.REQUIRED)
     public TaskPage loadAllTasks(final QueryParameters queryParameters) {
         TaskPage taskPage = new TaskPage();
-        long totalCount = blastTaskDao.getTasksCount(queryParameters.getFilters());
+        List<Filter> filters = queryParameters.getFilters();
```

Comment:
`filters` maybe an unmodifiable list (e.g. for call from `BlastTaskScheduledService`) so I'd suggest to create a new list in any case and add provided and default filter to it